### PR TITLE
Optimize queries

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: dash/dashblocks/templatetags/dashblocks.py [hunk 1]
```diff
@@ -55,18 +56,32 @@ def load_dashblocks(context, org, slug, tag=None):
...
         return getattr(settings, "DASHBLOCK_STRING_IF_INVALID", default_invalid) % slug
 
-    dashblocks = DashBlock.objects.filter(dashblock_type=dashblock_type, org=org, is_active=True)
-    dashblocks = dashblocks.order_by("-priority")
-
-    # filter by our tag if one was specified
-    if tag is not None:
-        dashblocks = dashblocks.filter(tags__icontains=tag)
+    if hasattr(dashblock_type, "prefetch_dashblocks"):
+        dashblocks = dashblock_type.prefetch_dashblocks
```
Comment (rowanseymour) [inline dash/dashblocks/templatetags/dashblocks.py:81]: in what scenario isn't this prefetched?

Comment (norkans7) [reply dash/dashblocks/templatetags/dashblocks.py:81]: None, I'll adjust that

File: dash/dashblocks/templatetags/dashblocks.py [hunk 2]
```diff
@@ -55,18 +56,30 @@ def load_dashblocks(context, org, slug, tag=None):
...
+            Prefetch(
+                "dashblock_set",
+                queryset=dashblocks_qs,
+                to_attr="prefetch_dashblocks",
+            )
+        )
+        .first()
+    )
+
+    if dashblock_type is None:
```
Comment (rowanseymour) [inline dash/dashblocks/templatetags/dashblocks.py:76]: As a general rule, don't write `is None` unless you specifically want to differentiate between falsey and `None`.. the thing you're checking might have an optimized `__bool__` method which won't be used if you're checking whether something is none or not

File: dash/dashblocks/templatetags/dashblocks.py (hunk 1 above)
Comment (rowanseymour) [reply dash/dashblocks/templatetags/dashblocks.py:82]: Still need to remove this `if` no?

**Pull request merged**